tests/atlas.phpt

```diff
@@ -0,0 +1,40 @@
+--TEST--
+Atlas Connectivity Tests
+--SKIPIF--
+<?php
+if (!file_exists('.travis.scripts/atlas-uris.txt')) { echo "skip Atlas URIs not found\n"; }
```

Comment:
We need to test for this, as in forks, this file can't be unencrypted (i.e., for Pull Requests)